src/flux2/constants.js

```diff
@@ -0,0 +1,8 @@
+export default {
```

Comment:
Are flux, flux2 and flux3 the same? I think you can remove the unused ones so it won't be confusing.